cmd/flags_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"flag"
+	"os"
+	"testing"
+)
+
+func TestParseFlags(t *testing.T) {
+	var a, b string
+	flag.StringVar(&a, "a", "", "")
+	flag.StringVar(&b, "b", "", "")
+
+	flag.CommandLine.Init(os.Args[0], flag.ContinueOnError)
+
+	tests := []struct {
```

Comment:
Please add a `name` field and include it in all of the `t.Errorf()` calls so that it's easy to identify which test case failed.